build/npm.js

```diff
@@ -42,10 +42,15 @@ const npx = async (...args) => {
     await executeProcess("npx", ...args);
 };
 
-async function npmInstall(name) {
+async function npmInstall(name, offLine = true) {
+    let args = ["--no-audit"];
+    if (offLine) {
+        args.push( "--prefer-offline" );
+    }
+    args.push( "ci");
```

Comment:
What's with the space after `(` - also below - does Prettier not complain?

Reply:
Er...